index.html

```diff
@@ -0,0 +1,161 @@
+    </head>
+    <body>
+    <main>
+       <header>
+        <nav class="navigation">
+            <div class="hamburger">
+                <span class="ham"></span>
+                <span class="ham"></span>
+                <span class="ham"></span>
+            </div>
+            <ul class="menu">
+                <li class="menu-items">
+                    <a href="./index.html">Home</a>
+                </li>
+                <li class="menu-items">
+                    <a href="./about.html">About</a>
+                </li>
+                <li class="menu-items">
+                    <a href="#">News</a>
+                </li>
+                <li class="menu-items">
+                    <a href="#">Contact</a>
+                </li>
+            </ul>
+            <div class="desktop-menu">
+                <ul class="nav-before">
+                    <li>
+                        <a href="#"><i class="bi bi-facebook"></i></a>
+                    </li>
+                    <li>
+                        <a href="#"><i class="bi bi-twitter"></i></a>
+                    </li>
+                    <li class="nav-icons">
+                        <a href="#" id="lang">English</a>
+                    </li>
+                    <li class="nav-icons">
+                        <a href="#">My Page</a>
+                    </li>
+                    <li class="nav-icons">
+                        <a href="#">Logout</a>
+                    </li>
+                </ul>
+                <ul class="desktop-menu-links">
+                    <li>
+                        <a href="./index.html">
+                         <img src="./icons/lg01.png" id="nav-logo" alt="cc logo">
+                        </a>
+                    </li>
+                    <li>
+                        <a href="./about.html">About</a>
+                    </li>
+                    <li>
+                        <a href="#about">Program</a>
+                    </li>
+                    <li>
+                        <a href="#">Live training</a>
+                    </li>
+                    <li>
+                        <a href="#">Sponsors</a>
+                    </li>
+                    <li>
+                        <a href="#">Web3 careers</a>
+                    </li>
+                    <li id="camp">
+                        <a href="#">CC Campaign</a>
+                    </ul>
+            </div>
+        </nav>
+       </header>
+        <section id="home" class="home-page">
+            <h1 class="headline"><span id="intro">Reorient, Rethink, Revolutionize</span>WELCOME<br>TO BLOCKCHAIN WORLD <br> AND NFT'S</h1>
+            <p class="summit-description">
+                Embrace Web3 with a range of certifications offered by the Blockchain Council, 

```

Comment:
- [ ] On the mobile menu, please make sure that this `<p>` element isn't overflowing the main programming section.
![Screenshot (5)](https://user-images.githubusercontent.com/47625407/182980499-62c5b42e-2406-447d-98a6-caa10f328b9a.png)